book_src/architecture/deserialize.md

```diff
@@ -29,7 +29,7 @@ Deserializers, like serializers, provide capabilities to objects during deserial
 don't need to bound their deserializers, but some like `Rc` require special traits in order to
 deserialize properly.
 
-The `Pooling` trait controls how pointers which were serialized shared are deserialized. Much like
+The `Pooling` trait controls how pointers which were serialized as shared are deserialized. Much like
```

Comment:
If you want to fix uses of which vs that in docs, then this should also be 'that' instead of 'which'.